pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -7447,7 +7932,9 @@ def solve_model(model, options=None):
                 model = postprocess_water_quality_calculation(model, opt)
         else:
             # option 2.1:
-            results = opt.solve(model, tee=True)
+            results = opt.solve(
+                model, tee=True, io_options={"add_options": ["GAMS_MODEL.optFile=1;"]}
```

Comment:
Please verify that all options are compatible with both GAMS and Gurobi to ensure that the model runs smoothly with either solver.